canary/e2e/features/liveness/face-detect.feature

```diff
@@ -8,4 +8,4 @@ Feature: Liveness Start Screen
   @react
   Scenario: Blazeface CDN is up
     Then I request "https://cdn.liveness.rekognition.amazonaws.com/face-detection/tensorflow/tfjs-backend-wasm/3.11.0/tfjs-backend-wasm-simd.wasm" and get "200"
-    And I request "https://cdn.liveness.rekognition.amazonaws.com/face-detection/tensorflow-models/blazeface/0.0.7/model/model.json" and get "200"
+    Then I request "https://cdn.liveness.rekognition.amazonaws.com/face-detection/tensorflow-models/blazeface/0.0.7/model/model.json" and get "200"
```

Comment:
Switch `and` with `then` bc `cypress-cucumber-preprocessor` dropped support for And since [v13](https://github.com/badeball/cypress-cucumber-preprocessor/releases/tag/v13.0.0)